src/lib/stitching/convection/schema.ts

```diff
@@ -22,6 +22,8 @@ export const executableConvectionSchema = () => {
     Asset: "ConsignmentSubmissionCategoryAsset",
     Category: "ConsignmentSubmissionCategoryAggregation",
     Offer: "ConsignmentOffer",
+    PageCursor: "ConsignmentPageCursor",
```

Comment:
Not sure about this, but I chose to include them as mapped names, as that's the approach Exchange is using. :cargocult:

Reply:
Yup, works 👍